src/OrbitGl/ThreadTrack.cpp

```diff
@@ -212,7 +212,8 @@ float ThreadTrack::GetDefaultBoxHeight() const {
 }
 
 Color ThreadTrack::GetTimerColor(const TimerInfo& timer_info, const internal::DrawData& draw_data) {
-  const uint64_t scope_id = app_->GetCaptureData().ProvideScopeId(timer_info);
+  const uint64_t scope_id =
+      app_->HasCaptureData() ? app_->GetCaptureData().ProvideScopeId(timer_info) : 0;
```

Comment:
Use `orbit_client_data::kInvalidScopeId` instead of `0`.

Reply:
Ah great, thanks, done.